package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "@colony/colony-js": "^6.1.0",
     "aws-amplify": "^4.3.43",
+    "decimal.js": "^10.4.3",
```

Comment:
This is really useful for ensuring consistency in data precision between front end and db.